clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go

```diff
@@ -79,6 +79,18 @@ func ListRuntimeObjectsForKind(c clientset.Interface, kind, namespace, labelSele
 			}
 			return nil
 		}
+	case "StatefulSet":
```

Comment:
Can you move this part to a separate PR - we want this merged no matter what will happen with this PR.